Adapt to Submissions update

siemensikkema opened the pull request:
- setup to setUp where necessary

steffendsommer commented on Sources/App/Models/Entities/AppUser/Extensions/AppUser+JWTKeychain.swift:
```diff
@@ -4,78 +4,74 @@ import Reset
-        static let readablePasswordKey = \UserLogin.password
-        static let readableUsernameKey = \UserLogin.email
+// MARK: - Creatable
+extension AppUser {
```
Any reason for why the conformance is not added to the type?

steffendsommer commented on Sources/App/Models/Entities/AppUser/Extensions/AppUser+JWTKeychain.swift:
```diff
@@ -4,78 +4,74 @@ import Reset
-        static let readablePasswordKey = \UserRegistration.password
-        static let readableUsernameKey = \UserRegistration.email
+// MARK: - Loginnable
+extension AppUser {
```
Any reason for why the conformance is not added to the type?

steffendsommer commented on Sources/App/Models/Entities/AppUser/Extensions/AppUser+JWTKeychain.swift:
```diff
@@ -4,78 +4,74 @@ import Reset
-        var username: String? {
-            return email
-        }
+// MARK: - PublicRepresentable
```
Any reason for why the conformance is not added to the type?

steffendsommer commented on Sources/App/Models/Entities/AppUser/Extensions/AppUser+JWTKeychain.swift:
```diff
@@ -4,78 +4,74 @@ import Reset
-        return req.future(UserPublic(email: email, name: name))
-    }
+// MARK: - Updatable
+extension AppUser {
```
Any reason for why the conformance is not added to the type?

steffendsommer commented on Sources/App/Models/Entities/AppUser/Extensions/AppUser+Submittable.swift:
```diff
@@ -4,78 +4,76 @@ import Submissions
+                    instance: instance,
                     label: "Password again",
-                    validators: [.count(8...)]
+                    validators: [Validator("") {
```
👍

steffendsommer commented on Sources/App/Setup/configure.swift:
```diff
@@ -67,12 +66,7 @@ public func configure(
-        try leafTags(config: &config, container)
-        return config
-    }
+    // Look at boot.swift
```
We're registering leaf tags in `Setup/leafTags.swift` (https://github.com/nodes-vapor/template/blob/feature/adapt-to-submissions/Sources/App/Setup/leafTags.swift) so this function needs to be called. Also, is it needed to do it through `boot.swift` still? I believe we did that due to the (now outdated) `MutableLeafTagContainer`.

siemensikkema replied:
do you have an example of how this should be done?

siemensikkema replied:
never mind 🙂

siemensikkema committed:
Adapt to Submissions update

- setup to setUp where necessary

Register leaf tags

Adapt to JWTKeychain and Submissions
